src/utils/suggestedResearchers.js

```diff
@@ -0,0 +1,29 @@
+import http from './httpService';
```

Comment:
Nice job. Is there a specific reason you abstracted the http request to the utils folder rather than the actions folder?

Reply:
Yes. To successfully get suggested researchers profile, I needed to do more logic and an extra call to the server. To avoid complexity in the actions, I had to abstract this logic